src/api/hooks/useSubmitTransaction.ts

```diff
@@ -43,7 +43,7 @@ const useSubmitTransaction = () => {
       network?.name.toLocaleLowerCase() !==
         (state.network_name === "local" ? "localhost" : state.network_name) &&
       // TODO: This is a hack to get around network being cached
-      wallet?.name !== "Google (AptosConnect)"
+      wallet?.name === "Google (AptosConnect)"
```

Comment:
So the reason for this hack is the Aptos Connect SDK handles network differently and there isnt an easy way to refresh the plugin. Thus in AptosConnect we are ignoring the network name check

Your change would make it so that explorer ignores the network name check for everything but Apto Connect. Which I think is not what you want?